src/main/java/com/amihaiemil/docker/Exec.java

```diff
@@ -46,4 +46,16 @@
      */
     JsonObject inspect() throws IOException, UnexpectedResponseException;
 
+    /**
+     * Return low-level information about this exec instance.
+     * @param containerId
+     * @param exec param of object
+     * @return ExecInstance object.
+     * @see <a href="https://docs.docker.com/engine/api/v1.35/#operation/ImageInspect">Inspect Image</a>
+     * @throws IOException If something goes wrong.
+     * @throws UnexpectedResponseException If the status response is not
+     *  the expected one (200 OK).
+     */
+    ExecInstance create(final String containerId, final JsonObject exec) throws IOException, UnexpectedResponseException;
```

Comment:
@jackpan123 I'm pretty sure this method should be in the [Execs](https://github.com/amihaiemil/docker-java-api/blob/master/src/main/java/com/amihaiemil/docker/Execs.java) interface.

Reply:
Only get method in Execs interface,  create method can provide execute the command line in a certain container.

Reply:
Could I change the return value of the method?